modules/24_trade/standard/equations.gms

```diff
@@ -0,0 +1,37 @@
+*** |  (C) 2006-2024 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+*** SOF ./modules/24_trade/standard/equations.gms
+
+*' @equations
+
+*** ---------------------------------------------------------------------------
+***        Trade Constraints
+*** ---------------------------------------------------------------------------
+
+***------------------------------------------------------
+*' Imports constrained by demand side of import country
+***------------------------------------------------------
+
+*Coal used in steel making is imported due to minimum quality requiremend (that is India)
+* active only after 2020; otherwise potential interference with constrained historic vm_Mport()
+* The constraint avoids Indian coal imports to drop to zero immediately
+
+q24_peimport_demandside(t,regi,enty)$(t.val gt 2020 AND sameas(regi, "IND") AND sameas(enty, "pecoal"))..
```

Comment:
Please, do not use any region in the GAMS code! You need to find a formulation that is independen of the region.

Reply:
Any proposal? Any precedent for orientation?